app/blueprints/assessments/templates/assessor_dashboard.html

```diff
@@ -75,116 +71,79 @@ <h2 class="govuk-heading-l fsd-banner-content"> {% if is_active_status %}All act
 {% block content %}
 {# CLOSE THE GOVUK-WIDTH-CONTAINER AND OPEN A WIDER CONTAINER FOR THE TABLE VIEW #}
 
+
 <div class="govuk-grid-row">
-    {% if migration_banner_enabled %}
+    {% if c %}
```

Comment:
Might be a stupid question, but what's "c" here?

Reply:
Not at all, its a good spot and i'm not sure how that one ended up there (I was debating removing this migration banner at some point but decieded to leace it for now as it might prove useful at some point in the future).